src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -325,7 +328,13 @@ private void AddRoslynOutputFilePath(ProjectInfo project, ProjectData projectDat
             var property = project.AnalysisSettings.FirstOrDefault(p => p.Id.EndsWith(".roslyn.reportFilePath"));
             if (property != null)
             {
-                projectData.RoslynReportFilePaths.Add(new FileInfo(property.Value));
+                var reportFilePaths = property.Value.Split(RoslynReportPathsDelimiter).Select(path => new FileInfo(path));
+                foreach (var reportFilePath in reportFilePaths)
+                {
+                    projectData.RoslynReportFilePaths.Add(reportFilePath);
+                }
+                // For compatibility with old SonarC#/VB.NET plugins we need to have only one path in this property
+                property.Value = projectData.RoslynReportFilePaths.First().FullName;
```

Comment:
Doesn't updating the property here mean that FixSarifReport will only be called for the first report?